srcjs/output_binding_html.js

```diff
@@ -174,6 +174,9 @@ function renderDependency(dep) {
         sheet.disabled = true;
         if (browser.isIE) sheet.cssText = "";
         $(sheet.ownerNode).remove();
+        $(document).trigger({
+          type: 'shiny:restyle', oldSheet: sheet
```

Comment:
What is `oldSheet` used for?

Reply:
Mainly so can get at the `href` so you which `<link>` was updated

Reply:
That being said, maybe we could add the `name` and `version` of the relevant HTML dependency